src/elements/form/controls/BaseControl.ts

```diff
@@ -38,6 +38,7 @@ export interface NovoControlConfig {
     description?: string;
     tooltip?: string;
     tooltipPosition?: string;
+    layoutOptions?: any;
```

Comment:
Instead of any, make this like `layoutOptions?: { order?: string, download?:boolean ... }` that way the implementer knows how to set them